packages/react-components/src/components/Modal/Modal.stories.tsx

```diff
@@ -51,7 +53,15 @@ const StoryTemplate: StoryFn<ModalProps> = ({
   const [isOpen, setIsOpen] = React.useState(true);
 
   return (
-    <>
+    <div
+      style={{
```

Comment:
please move it to const, outside the component's body. Every inline object or a functions are recreated on each render and we do not want that :)